tests/Graph/SubGraphDeployment/auxiliary-verifier.ts

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env node
+
+import * as commander from 'commander';
+import { SubscriptionClient } from "subscriptions-transport-ws";
+import WebSocket = require('ws');
+
+import Logger from "../../../src/Logger";
+
+let mosaic = commander
+  .arguments('<auxiliary-chain-identifier> <graph-ws-port>');
+
+mosaic.action(
```

Comment:
Should this file be simple typescript file? is there really a need of commander here? 
```
/node_modules/.bin/ts-node tests/Graph/SubGraphDeployment/origin-verifier.ts
```

Reply:
commander gives us more power to handle cli params. Without which we might have to use args[0] etc. Thus would suggest to keep this.